integr8ly/tutorial-web-app
# Update PatternFly version and Red Hat branding

## Motivation
Update PatternFly dependencies to use the latest versions. Includes the breaking change from 1.x to 2.x, which is the driver for this large PR.

## What
Update PatternFly Core to 2.6.11 and PatternFly React-Core to 3.16.10.

## Why
Implement and test the breaking changes introduced with PatternFly v2.x.

## How
Update `package.json` file, run app locally and verify that UI is functional. Any broken aspects were fixed.

## Verification Steps

1. Pull PR locally
2. Install latest updates (`yarn install`) and rebuild the app (`yarn build`)
3. Run the app and go through a Walkthrough
4. View Provisioning screen to see the new background w/watermark
5. Open the About Modal to see the new background w/watermark
6. Finish a Walkthrough to see the Congratulations screen with new background w/watermark and colors.

## Checklist:


## Progress


## Additional Notes

<img width="1920" alt="Screen Shot 2019-05-21 at 10 36 49 AM" src="https://user-images.githubusercontent.com/4032718/58105409-793ecf80-7bb4-11e9-8eab-04ae28ec0e0c.png">
<img width="1920" alt="Screen Shot 2019-05-21 at 10 36 59 AM" src="https://user-images.githubusercontent.com/4032718/58105418-7cd25680-7bb4-11e9-9477-c87b6244cf00.png">
<img width="1920" alt="Screen Shot 2019-05-21 at 10 37 29 AM" src="https://user-images.githubusercontent.com/4032718/58105435-81970a80-7bb4-11e9-9342-7a04cb9ba187.png">
<img width="1920" alt="Screen Shot 2019-05-21 at 10 37 20 AM" src="https://user-images.githubusercontent.com/4032718/58105444-852a9180-7bb4-11e9-8f04-9fc9a6d4ab52.png">
<img width="1920" alt="Screen Shot 2019-05-21 at 10 37 09 AM" src="https://user-images.githubusercontent.com/4032718/58105457-88258200-7bb4-11e9-947d-603b6a3787a7.png">
